src/pages/Main/RegionButton/RegionModal/RegionModal.js

```diff
@@ -0,0 +1,115 @@
+import React, { useState, useEffect, useRef } from 'react';
+import styled from 'styled-components';
+
+function Modal({ title, handleModal, tagFetchHandler }) {
+  return (
+    <ModalBackgournd>
+      <ModalOverlay onClick={() => handleModal()} />
+      <ModalBox>
+        <ModalHeader>
+          <ResetButton />
+          <h1>{title}</h1>
+          <CloseButton name="close" onClick={() => handleModal()}>
```

Comment:
```suggestion
          <CloseButton name="close" onClick={handleModal}>
```
불필요한 inline 함수 선언입니다! 다른 부분도 마찬가지입니다!

Reply:
랜더링될때 실행되던 기억이 있어서 습관적으로 저렇게 사용했내요 ㅠㅠ

Reply:
수정해주세요!